Airwallex/Payment/Sources/CardPaymentConsentSectionController.swift

```diff
@@ -0,0 +1,183 @@
+        }
+        let viewModel = PaymentConsentCellViewModel(
+            image: image,
+            text: "\(brand.capitalized) •••• \(card.last4 ?? "")",
+            buttonAction: { [weak self] in
+                self?.showAlertForDelete(consent, indexPath: indexPath)
+            }
+        )
+        cell.setup(viewModel)
+        return cell
+    }
+    
+    func supplementaryView(for collectionView: UICollectionView, ofKind elementKind: String, at indexPath: IndexPath) -> UICollectionReusableView {
+        let header = collectionView.dequeueReusableSupplementaryView(ofKind: elementKind, withReuseIdentifier: CardPaymentSectionHeader.reuseIdentifier, for: indexPath) as! CardPaymentSectionHeader
+        let viewModel = CardPaymentSectionHeaderViewModel(
+            title: NSLocalizedString("Choose a card", comment: ""),
+            actionTitle: NSLocalizedString("Add new", bundle: .payment, comment: ""),
+            buttonAction: { [weak self] in
+                guard let self else { return }
+                // wpdebug
+                do {
+                    var snapshot = self.context.currentSnapshot()
+                    snapshot.reloadSections([self.section])
+                    context.dataSource.apply(snapshot)
+                    return
+                }
+                
+                showTODO()
+            }
+        )
+        header.setup(viewModel)
+        return header
+    }
+    
+    func layout(environment: any NSCollectionLayoutEnvironment) -> NSCollectionLayoutSection {
+        let itemSize = NSCollectionLayoutSize(
+            widthDimension: .fractionalWidth(1),
+            heightDimension: .fractionalHeight(1)
+        )
+        let item = NSCollectionLayoutItem(layoutSize: itemSize)
+        
+        let groupSize = NSCollectionLayoutSize(
+            widthDimension: .fractionalWidth(1),
+            heightDimension: .absolute(56)
+        )
+        let group = NSCollectionLayoutGroup.vertical(layoutSize: groupSize, subitems: [item])
+        let section = NSCollectionLayoutSection(group: group)
+        section.contentInsets = .init(top: .spacing_16, leading: .spacing_16, bottom: .spacing_16, trailing: .spacing_16)
+        
+        let headerSize = NSCollectionLayoutSize(widthDimension: .fractionalWidth(1), heightDimension: .estimated(32))
+        let header = NSCollectionLayoutBoundarySupplementaryItem(
+            layoutSize: headerSize,
+            elementKind: UICollectionView.elementKindSectionHeader,
+            alignment: .top
+        )
+        section.boundarySupplementaryItems = [header]
+        return section
+    }
+    
+    func registerReusableViews(to collectionView: UICollectionView) {
+        collectionView.register(
+            CardPaymentSectionHeader.self,
+            forSupplementaryViewOfKind: UICollectionView.elementKindSectionHeader,
+            withReuseIdentifier: CardPaymentSectionHeader.reuseIdentifier
```

Comment:
Can we have a extension method for collectionView so that we don't need to call '.reuseIdentifier' every time?

extension UICollectionView {
    func register(_ cellClass: any ViewReusable.Type) {
        register(cellClass, forCellWithReuseIdentifier: cellClass.reuseIdentifier)
    }
}

Reply:
done